security/certs.go

```diff
@@ -28,6 +28,28 @@ import (
 	"github.com/cockroachdb/cockroach/util"
 )
 
+// loadCACertAndKey loads the certificate and key files in the specified
+// directory, parses them, and returns the x509 certificate and private key.
+func loadCACertAndKey(certsDir string) (*x509.Certificate, crypto.PrivateKey, error) {
```

Comment:
oh. I was reading "why does it not return a pointer".
This is ultimately what x509.CreateCertificate expects. I figured I might as well return a pointer from the get go.